modules/osd-monitoring-configurable-monitoring-components.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-configuring-the-monitoring-stack.adoc
+
+[id="configurable-monitoring-components_{context}"]
+= Configurable monitoring components
+
+This table shows the monitoring components you can configure and the keys used to specify the components in the `user-workload-monitoring-config` `ConfigMap` objects:
+
+.Configurable monitoring components
+[options="header"]
+|====
+//|Component |cluster-monitoring-config ConfigMap key |user-workload-monitoring-config config map key
+|Prometheus Operator |`prometheusOperator` |`prometheusOperator`
+|Prometheus |`prometheusK8s` |`prometheus`
+//|Alertmanager |`alertmanagerMain` |
+//|kube-state-metrics |`kubeStateMetrics` |
+//|openshift-state-metrics |`openshiftStateMetrics` |
+//|Grafana |`grafana` |
+//|Telemeter Client |`telemeterClient` |
+//|Prometheus Adapter |`k8sPrometheusAdapter` |
+//|Thanos Querier |`thanosQuerier` |
+|Thanos Ruler | |`thanosRuler`
+|====
+
+//[NOTE]
+====
+The Prometheus key is called `prometheusK8s` in the `cluster-monitoring-config` `ConfigMap` object and `prometheus` in the `user-workload-monitoring-config` `ConfigMap` object.
```

Comment:
I don't think we need to keep the NOTE, since there is not `cluster-monitoring-config` `ConfigMap` mentioned in the doc

Reply:
The note was commented out, but I've removed it entirely.